car_framework/car_service.py

```diff
@@ -334,11 +334,11 @@ def setup_extension(self, extension):
             'version': extension.version,
             'schema': json.loads(extension.schema)
         }
-        r = self.communicator.post(CAR_SCHEMA, data=json.dumps(data), api_version='/api/car/v3')
+        r = self.communicator.post(CAR_SCHEMA, data=json.dumps(data), api_version='/v3')
         if r.status_code not in (200, 201):
             raise Exception('Error when posting schema extension: %d' % r.status_code)
 
-    def get_import_schema(self, api_version='/api/car/v3', version='v2'):
+    def get_import_schema(self, api_version='/v3', version='v2'):
```

Comment:
Just to confirm this change is correct as opposed to setting the v2 version to `auth_token/v2` ? Also should the v2 have the backslash at the beginning like v3?

Reply:
So, they'll send `/api/car/v2` or `/api/car/auth_token/v2` depending on edge or cluster. They fix is to find  `v2` instead of `/api/car/v2` and replace it with `v3`.

No, `version ` is a parameter passed. Its not in path